src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,39 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User register;
+
+        try {
+            register = authenticationService.register("kyrylh@testmail.com", "2134");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Can't register user", e);
+        }
+
+        System.out.println("Registered user: " + register);
+        User login;
+
+        try {
+            login = authenticationService.login("kyrylh@testmail.com", "2134");
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Can't login user", e);
+        }
+        System.out.println(login);
+
+        try {
+            login = authenticationService.login("kyrylh@testmail.com", "6365");
```

Comment:
You can transfer this login to previous try/catch block to prevent code dublicating